docs/how-to/freeze-checkbox-version.rst

```diff
@@ -0,0 +1,218 @@
+and freeze it in the current version:
+
+.. code-block:: bash
+
+   $ snap refresh --hold=forever checkbox checkbox22
+
+To check which snaps are being held, you can look for ``held`` in the ``notes``
+column when running ``snap list``:
+
+.. code-block::
+
+   $ snap list
+   Name        Version        Rev    Tracking       Publisher            Notes
+   checkbox    3.3.0-dev19    5224   22.04/stable   ce-certification-qa  classic,held
+   checkbox22  3.3.0-dev19    726    latest/stable  ce-certification-qa  held
+
+To remove the hold, just run the ``snap refresh --unhold`` command for each
+snap.
+
+.. code-block::
+
+   $ snap refresh --hold=forever checkbox checkbox22
+   $ snap list
+   Name        Version        Rev    Tracking       Publisher            Notes
+   checkbox    3.3.0-dev19    5224   22.04/stable   ce-certification-qa  classic
+   checkbox22  3.3.0-dev19    726    latest/stable  ce-certification-qa  -
+
+How to freeze Checkbox Debs
+============================
+
+Using Ubuntu snapshot service (20.04 or later)
+----------------------------------------------
+
+To freeze the Checkbox version of a deb package we will make use of the Ubuntu
+snapshot service. This service allows you to see and use the Ubuntu archive, as
+well as all PPAs (both private and public), as it was at any specified date and
+time, for any time and date after 1 March 2023.
+
+Snapshots are supported in Ubuntu 23.10 or later, and also on updated
+installations of Ubuntu 20.04 LTS (with ``apt`` 2.0.10) and Ubuntu 22.04
+LTS(with ``apt`` 2.4.11).
+
+Enable the snapshot Service
+```````````````````````````
+
+For Ubuntu 24.04
+''''''''''''''''
+
+The apt included in Ubuntu 24.04 and later automatically detects when snapshots
+are supported for a repository and are enabled by default.
+
+Nevertheless, for PPAs in Launchpad it is sometimes necessary to modify the URI
+field to match the URL of the snapshot. In the case of checkbox, you have to
+edit
+``/etc/apt/sources.list.d/checkbox-dev-ubuntu-{ppa-name}-{ubuntu-version}.list``
+and add a tilde before checkbox-dev.
+
+For example:
+
+.. code-block::
+
+   # In “/etc/apt/sources.list.d/checkbox-dev-ubuntu-beta-noble.list”
+   Types: deb
+   URIs: https://ppa.launchpadcontent.net/~checkbox-dev/beta/ubuntu/
+   Suites: noble
+   Components: main
+   Signed-By: {KEY} 
+
+For Ubuntu 23.10 and earlier
+''''''''''''''''''''''''''''
+On Ubuntu 23.10 and earlier, edit
+``/etc/apt/sources.list.d/checkbox-dev-ubuntu-{ppa-name}-{ubuntu-version}.list``
+and add ``[snapshot=yes]`` into the standard prefix. You also need to modify the
+PPA URL to include the tilde before checkbox-dev so it points to the correct
+snapshot URL. For example:
+
+.. code-block::
+
+   # In “/etc/apt/sources.list.d/checkbox-dev-ubuntu-beta-jammy.list”
+   deb [snapshot=yes] https://ppa.launchpadcontent.net/~checkbox-dev/beta/ubuntu jammy main
+
+
+Use a Snapshot ID with apt Commands
```

Comment:
```suggestion
Use a Snapshot ID with apt commands
```

Also, add a blank line under the back ticks (I cannot put this in the suggestion because it confuses GitHub!)